test/chainsaw/testmetrics/metrics-hpa/00-install.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podtato-head-entry
+  labels:
+    app: podtato-head
+spec:
+  selector:
+    matchLabels:
+      component: podtato-head-entry
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: podtato-head-entry
+    spec:
```

Comment:
this pr excludes sonarcloud checks from the tests folders... this is reasonable because otherwise, the test configs would be too lengthy and difficult to understand I will add some of the items here to our TODO ticket :)